roles/kubernetes_setup/files/scheduler-pod.json

```diff
@@ -0,0 +1,39 @@
+{
+  "kind": "Pod",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "kube-scheduler"
+  },
+  "spec": {
+    "hostNetwork": true,
+    "containers": [
+      {
+        "name": "kube-scheduler",
+        "image": "registry.access.redhat.com/rhel7/kubernetes-scheduler",
```

Comment:
Oh, I remember the reason for containerized kube:  it's what is used in the RHEL docs.  This playbook is trying to replicate those instructions.